README.md

```diff
@@ -1,455 +1,30 @@
-<div align="center">
-
-# Text Embeddings Inference
-
-<a href="https://github.com/huggingface/text-embeddings-inference">
-  <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/huggingface/text-embeddings-inference?style=social">
-</a>
-<a href="https://huggingface.github.io/text-embeddings-inference">
-  <img alt="Swagger API documentation" src="https://img.shields.io/badge/API-Swagger-informational">
-</a>
-
-A blazing fast inference solution for text embeddings models.
-
-Benchmark for [BAAI/bge-base-en-v1.5](https://huggingface.co/BAAI/bge-base-en-v1.5) on an Nvidia A10 with a sequence
-length of 512 tokens:
-
-<p>
-  <img src="assets/bs1-lat.png" width="400" />
-  <img src="assets/bs1-tp.png" width="400" />
-</p>
-<p>
-  <img src="assets/bs32-lat.png" width="400" />
-  <img src="assets/bs32-tp.png" width="400" />
-</p>
-
-</div>
-
-## Table of contents
-
-- [Get Started](#get-started)
-    - [Supported Models](#supported-models)
-    - [Docker](#docker)
-    - [Docker Images](#docker-images)
-    - [API Documentation](#api-documentation)
-    - [Using a private or gated model](#using-a-private-or-gated-model)
-    - [Using Re-rankers models](#using-re-rankers-models)
-    - [Using Sequence Classification models](#using-sequence-classification-models)
-    - [Distributed Tracing](#distributed-tracing)
-    - [gRPC](#grpc)
-- [Local Install](#local-install)
-- [Docker Build](#docker-build)
-
-Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence
-classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding,
-Ember, GTE and E5. TEI implements many features such as:
-
-* No model graph compilation step
-* Metal support for local execution on Macs
-* Small docker images and fast boot times. Get ready for true serverless!
-* Token based dynamic batching
-* Optimized transformers code for inference using [Flash Attention](https://github.com/HazyResearch/flash-attention),
-  [Candle](https://github.com/huggingface/candle)
-  and [cuBLASLt](https://docs.nvidia.com/cuda/cublas/#using-the-cublaslt-api)
-* [Safetensors](https://github.com/huggingface/safetensors) weight loading
-* Production ready (distributed tracing with Open Telemetry, Prometheus metrics)
-
-## Get Started
-
-### Supported Models
-
-#### Text Embeddings
-
-You can use any JinaBERT model with Alibi or absolute positions or any BERT, CamemBERT, RoBERTa, or XLM-RoBERTa model
-with absolute positions in `text-embeddings-inference`.
-
-**Support for other model types will be added in the future.**
-
-Examples of supported models:
-
```

Comment:
can we also have the supported models like below?

Reply:
Sure, what about doing it in the same commit that will upgrade transformers? This way we will retest new transformers package and double check supported models at the same time.